test_runner/regress/test_lfc_prewarm.py

```diff
@@ -90,9 +100,10 @@ def test_lfc_prewarm(neon_simple_env: NeonEnv, query: LfcQueryMethod):
     lfc_conn = endpoint.connect(dbname="lfc")
     lfc_cur = lfc_conn.cursor()
 
-    if query is LfcQueryMethod.COMPUTE_CTL:
+    # if autoprewarm = on, endpoint is prewarmed when available for connections
```

Comment:
Is it true? Autoprewarm at start mustn't be blocking. And I look at the code in `prewarm_lfc`. It seems to spawn a thread without waiting for it